### PR TITLE
Adding accessibility

### PR DESCRIPTION
**Pull request by EricMiddie**
Updated content descriptions for all relevant buttons (I think).

Used the Layout Inspector to make sure that all were correctly applied

### CODE REVIEW

File: app/src/main/java/com/example/dungeontest/composables/MultiFloatingActionButton.kt [hunk 1]
```diff
@@ -75,7 +75,7 @@ fun MultiFloatingActionButton(items: List<MiniFabItems>) {
                 .rotate(rotation),
             containerColor = MaterialTheme.colorScheme.primary,
         ) {
-            Icon(imageVector = Icons.Default.Add, contentDescription = "")
+            Icon(imageVector = Icons.Default.Add, contentDescription = "Button to Save, Add, Edit and Remove Rooms")
```
Comment (BareBeaverBat) [inline app/src/main/java/com/example/dungeontest/composables/MultiFloatingActionButton.kt]:
```suggestion
            Icon(imageVector = Icons.Default.Add, contentDescription = "Button to Save Map, Rename Rooms, or Add/Remove Hallways")
```

**Review by BareBeaverBat — CHANGES_REQUESTED** [review]
looks good, 1 clarification requested

File: app/src/main/java/com/example/dungeontest/composables/MultiFloatingActionButton.kt (hunk 1 above)
Comment (pepsipotty) [reply app/src/main/java/com/example/dungeontest/composables/MultiFloatingActionButton.kt]: Concerned that TTS doesnt handle the "/" character all that well

Comment (BareBeaverBat) [reply app/src/main/java/com/example/dungeontest/composables/MultiFloatingActionButton.kt]: Good point
```suggestion
            Icon(imageVector = Icons.Default.Add, contentDescription = "Button to Save Map, Rename Room, and Add or Remove Hallway")
```

**Review by pepsipotty — APPROVED** [review]

**Review by pepsipotty — APPROVED** [review]

**Review by BareBeaverBat — APPROVED** [review]
ty

**Pull request merged**